sources/Inputs/include/SimulationParams.h

```diff
@@ -0,0 +1,124 @@
+//
+// Copyright (c) 2021, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+
+/**
+ * @file  SimulationParams.h
+ *
+ * @brief Manager of simulation parameters
+ *
+ */
+
+#pragma once
+
+#include "Configuration.h"
+#include "Contingencies.h"
+#include "Log.h"
+#include "Options.h"
+#include "version.h"
+
+#include <DYNError.h>
+#include <DYNInitXml.h>
+#include <DYNIoDico.h>
+#include <DYNMPIContext.h>
+#include <boost/filesystem.hpp>
+#include <boost/property_tree/json_parser.hpp>
+#include <boost/property_tree/ptree.hpp>
+#include <chrono>
+#include <cstdlib>
+#include <sstream>
+
+/**
+ * @brief Simulation parameters
+ *
+ * This sructure aims at grouping in one place options needed by the simulation
+ * to run.
+ */
+struct SimulationParams {
+  std::string locale;                                               ///< locale
+  boost::filesystem::path networkFilePath;                          ///< path to network file
+  boost::filesystem::path resourcesDirPath;                         ///< path to resource directory
+  boost::filesystem::path parametersDirPath;                        ///< path to parameters directory
+  DYNAlgorithms::mpi::Context const* mpiContext;                    ///< mpi context
```

Comment:
mpiContext is a singleton accessible from everywhere, do we really need to store it here?